docs: on Python update (#285)
GreptimeTeam/docs: docs/user-guide/python-coprocessor/overview.md

# Note:
The Python coprocessor is currently in its experimental phase, and the API may undergo some changes.

> Using coprocessor with CPython also involves some complications. You must set up the correct Python shared library, which can be a bit challenging. But you can also use RustPython which doesn't require additional setup.

Reviewer: ```suggestion
Using coprocessor with CPython also involves some complications. You must set up the correct Python shared library, which can be a bit challenging. But you can also use RustPython which doesn't require additional setup.

```

> The recommended way is to use `conda` for managing your Python environment, you must create a Python environment with the same version of Python as the binary you download requires(Or just use docker container) and run the `greptime` binary inside it. A less recommended way is to install the exact version of python manually, and set the `LD_LIBRARY_PATH` environment variable to the directory where the `libpython<VERSION>.so`(the number of <VERSION> depends on the version of Python where ) file is located.

Reviewer: ```suggestion
The recommended way is to utilize `conda` for managing your Python environment. Firstly, create a Python environment with the same version of Python demanded by the binary you download. Alternatively, you can employ a docker container and execute the `greptime` binary within it.

A less recommended way is to manually install the exact version of Python required and set the `LD_LIBRARY_PATH` environment variable to the directory containing the `libpython<VERSION>.so` file. The version number of `<VERSION>` varies according to the version of Python being used.
```

> There is two Backend for Python Coprocessor:

Reviewer: ```suggestion
There is two Backend for Python Coprocessor:

```

> 1. RustPython Interpreter: this is supported without need to install any python library, but it is not as fast as CPython Backend. Release Binary without `pyo3` in its name is using RustPython Interpreter. You can still use Python Syntax in RustPython Interpreter, but you can't use any third-parties libs.

Reviewer: ```suggestion
1. RustPython Interpreter: this is supported without installing any Python library, but it is not as fast as CPython Backend. The Release Binary without `pyo3` in its name uses RustPython Interpreter. While you can still use Python Syntax in RustPython Interpreter,  you can't use any third-parties libs.
```

> 2. CPython Interpreter: this is the usual and most common python, using it allows you to use all sort of third-parties libs, but you need to install the correct Python shared library. Release Binary with `pyo3` in its name is using CPython Interpreter.

Reviewer: ```suggestion
2. CPython Interpreter: this is the most commonly used version of Python. It allows you to use all sorts of third-parties libs, but you need to install the correct Python shared library. Any Release Binary with `pyo3` in its name uses CPython Interpreter.
```
